bot/src/main/resources/application.yml

```diff
@@ -1,6 +1,22 @@
 app:
   telegram-token: ${TELEGRAM_API_KEY}
 
+
+retry:
+  retry-type: exponential
+  duration: 5
+  attempts: 2
+  codes: 500, 501, 502, 503, 504, 521, 522, 523, 524
```

Comment:
https://stackoverflow.com/questions/51770071/what-are-the-http-codes-to-automatically-retry-the-request